crates/parry3d-f64/Cargo.toml

```diff
@@ -44,6 +44,7 @@ simd-stable = ["simba/wide", "simd-is-enabled"]
 simd-nightly = ["simba/portable_simd", "simd-is-enabled"]
 enhanced-determinism = ["simba/libm_force", "indexmap"]
 parallel = ["rayon"]
+# Adds `TriMesh:to_obj_file` function.
 wavefront = ["obj"]
```

Comment:
Adding documentation on `cargo.toml` is idiomatic to rust ecosystem (see serde or bevy)